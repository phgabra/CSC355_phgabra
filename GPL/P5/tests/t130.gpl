int nums[5];
int i = 2;
int j = 40;
int k = j + nums[i];
