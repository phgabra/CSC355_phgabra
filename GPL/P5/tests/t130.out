gpl.cpp::main()
  input file(tests/t130.gpl)
  random seed(42)
  read_keypresses_from_standard_input(false)
  dump_pixels(false)
  symbol_table(true)
  graphics(false)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().

No errors found (parser probably worked correctly).


Printing the symbol table from main()
int i = 2
int j = 40
int k = 40
int nums[0] = 0
int nums[1] = 0
int nums[2] = 0
int nums[3] = 0
int nums[4] = 0
Graphics is turned off by the Makefile.  Program exiting.

gpl.cpp::main() done.
